apps/store/src/utils/useAllowActiveStylesInSafari.ts

```diff
@@ -0,0 +1,10 @@
+import { useEffect } from 'react'
+
+// Source: https://css-tricks.com/snippets/css/remove-gray-highlight-when-tapping-links-in-mobile-safari/
+export const useAllowActiveStylesInSafari = () => {
+  useEffect(() => {
+    const handler = () => {}
+    document.addEventListener('touchstart', handler, true)
+    return () => document.removeEventListener('touchstart', handler, true)
+  })
```

Comment:
Shouldn't be an effect with empty array as the dependency list? 🤔 

```suggestion
export const useAllowActiveStylesInSafari = () => {
  useEffect(() => {
    const handler = () => {}
    document.addEventListener('touchstart', handler, true)
    return () => document.removeEventListener('touchstart', handler, true)
  }, [])
```